torch/csrc/jit/passes/onnx/constant_map.cpp

```diff
@@ -205,20 +205,12 @@ c10::optional<c10::SymbolicShape> ConstantValueMap::GetShapeValue(
   return ConstantValueMap::getInstance().shapeValueMap[tensorName];
 }
 
-void ConstantValueMap::SetGeneratedShape(std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto> generated_shape) {
-  ConstantValueMap::getInstance().generatedShapeDataByName = generated_shape;
-}
-
-std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto> ConstantValueMap::GetGeneratedShape() {
+std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto>& ConstantValueMap::GetGeneratedShapeDataByName() {
```

Comment:
Thanks for making the naming consistent. I am sorry for not expressing an opinion on this earlier, but now I think this name is quite long.
Would we lose any important info if we rename it to `GetGeneratedShapes`?

Please also add a comment here that explains what it's returning. In particular what the word `Generated` refers to.

Reply:
Or maybe to match the ONNX function name `GetInferredShapeData` / `SetInferredShapeData`?

Reply:
Yes, I initially used `GetGeneratedShape` to prevent longer name. However, now `GetInferredShapeData` looks clearer to me so I will use it with a comment for this function.

Reply:
Maybe also type alias `std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto>` to make it shorter

Reply:
I chose to use `using ShapeDataMap = std::unordered_map<std::string, ::ONNX_NAMESPACE::TensorShapeProto>;`. Thanks for the suggestion!